src/openvic-simulation/scripts/Condition.cpp

```diff
@@ -6,6 +6,10 @@
 using namespace OpenVic;
 using namespace OpenVic::NodeTools;
 
+using enum value_type_t;
```

Comment:
Isn't this what spartan calls "polluting the global scope"?

Reply:
This is inside a `cpp` file, so it doesn't affect any other file. Almost every single function in that file uses those 3 enums, it's no more polluting than the two `namespace` includes above (actually with fewer potential negative consequences, since an enum can't contain functions).